packages/calcite-components/src/components/table/table.e2e.ts

```diff
@@ -3178,4 +3178,122 @@ describe("keyboard navigation", () => {
       );
     });
   });
+
+  describe("pagination", () => {
+    it("starts on specific page and programmatically changes page", async () => {
```

Comment:
Could we add a test that updates an invalid `current-page` at load? Maybe have it set to `current-page="100"` with only a few rows present, to ensure it's being set to the last available set.